system-int-test/test/cmr/system_int_test/ingest/provider_ingest_permissions_test.clj

```diff
@@ -46,21 +46,12 @@
         prov-admin-update-group-concept-id (echo-util/get-or-create-group
                                             (s/context)
                                             "prov-admin-update-group")
-        prov-admin-update-group-concept-id-np (echo-util/get-or-create-group
```

Comment:
These -np were added so that we can have different user tokens. Now we the permission for a user is provider dependent in the cache, we don't need to distinguish the permissions through different user tokens.